notebooks/community/neo4j/graph_paysim.ipynb

```diff
@@ -1,1138 +1,1121 @@
 {
```

Comment:
<p>Instead of "AutoML Tables"  which uses the legacy Cloud AI Platform language, can you please use "AutoML Tabular Data" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/599/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>